subcommands/create-database

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+source "$(dirname "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)")/config"
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
```

Comment:
also add `-u` to `set` as per https://coderwall.com/p/fkfaqq/safer-bash-scripts-with-set-euxo-pipefail

Reply:
Didn't know about it. While this line was copy-pasted from `subcommands/create`, I think, I'll add -u to set.